src/app/machines/views/MachineList/MachineList.tsx

```diff
@@ -103,6 +121,24 @@ const MachineList = ({
         />
       ) : null}
       {!headerFormOpen ? <ErrorsNotification errors={machinesErrors} /> : null}
+      {configuredControllers.length >= 1 &&
+      unconfiguredControllers.length >= 1 ? (
+        <Notification severity="caution" title="Incomplete Vault integration">
+          Configure {unconfiguredControllers.length} other{" "}
+          <a href="/controllers">
```

Comment:
Could you please use a `Link` component instead of a basic anchor tag. We use a standard `a` only for external links (anything outside of react-router).

```tsx
import { Link } from "react-router-dom-v5-compat";
import urls from "app/base/urls";
```

```suggestion
          <Link to={urls.controllers.index}>
```